apps/user/base.py

```diff
@@ -21,6 +21,9 @@ def create_superuser(self, **kwargs):
         user.save(using=self._db)
         return user
 
+    def normalize_email(self, email):
```

Comment:
I'm not super familiar with how this method is used throughout Django, but did find this: https://docs.djangoproject.com/en/3.0/topics/auth/customizing/#django.contrib.auth.models.AbstractUser.clean

> Normalizes the email by calling BaseUserManager.normalize_email(). If you override this method, be sure to call super() to retain the normalization.

Reply:
Ah yes, we are not inheriting `clean()` from AbstractUser because we're subclassing AbstractBaseUser, which has a different `clean()` that serves inscrutable purposes. Fixed in 
d5bafb9.

Looks like `authtools` also overlooks this and thus does not take advantage of Django's default email normalization (lowercasing the domain only) - I'll make a note of this for them, too.